src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Services/UriHelper.ts

```diff
@@ -28,8 +28,11 @@ function enableNavigationInterception(assemblyName: string, functionName: string
       const href = anchorTarget.getAttribute(hrefAttributeName)!;
       const absoluteHref = toAbsoluteUri(href);
 
+      // https://github.com/aspnet/Blazor/issues/1352#issuecomment-415444870
+      const isExternal = ['_blank', '_parent', '_self', '_top'].indexOf(anchorTarget.getAttribute('target') || '') > -1;
+
       // Don't stop ctrl/meta-click (etc) from opening links in new tabs/windows
-      if (isWithinBaseUriSpace(absoluteHref) && !eventHasSpecialKey(event)) {
+      if (isWithinBaseUriSpace(absoluteHref) && !eventHasSpecialKey(event) && !isExternal) {
```

Comment:
This logic doesn't deal with other external targets, e.g., the custom names of popup windows opened previously.

Could we simplify the rule to "skip interception if there's a `target` attribute with any value other than `_self`"? That covers all the cases AFAIK.

Could be:

```typescript
const targetAttributeValue = anchorTarget.getAttribute('target');
const opensInSameFrame = !targetAttributeValue || targetAttributeValue === '_self';

if (isWithinBaseUriSpace(absoluteHref) && !eventHasSpecialKey(event) && opensInSameFrame) {
   ...
}
```

Reply:
Also we don't need the comment that links to the GitHub issue.

Reply:
In this case, I can not open the file in the current tab.
`<a href="/file.pdf" target="_self">Open file this tab</a>`

**Error**: https://github.com/aspnet/Blazor/issues/1367

Reply:
Given that any explicit **target="_*"** value should be ignored by the router, you can simplify the code to this state.

```
const targetAttributeValue = anchorTarget.getAttribute('target');

if (isWithinBaseUriSpace(absoluteHref) && !eventHasSpecialKey(event) && !targetAttributeValue ) {
   ...
}
```

Reply:
That sounds like you're trying to use `target="_self"` as a hack to skip click interception. I don't think that's the right way to indicate that you don't want to intercept the click, because `target="_self"` doesn't have any special behavior like that according to HTML specs.

Rather, if there is to be a way of indicating that a given link should open in the same frame but not do click interception, that would be a separate feature.

Reply:
There's a good chance we'll end up with a feature similar to your proposed `data-route="skip"`, but could we leave that to a separate PR?

Generally we find it good to keep different features in different PRs. So ideally this PR will handle only the `target` attribute handling.